### PR TITLE
Replace glew with GLES3 for linux aarch64

### PR DESCRIPTION
**Pull request by jpd002**

### CODE REVIEW

File: include/opengl/OpenGlDef.h [hunk 1]
```diff
@@ -33,6 +33,12 @@
 
 #endif
 
+#elif defined(__arch64__) && defined(__linux__)
```
Comment (mrcmunir) [inline include/opengl/OpenGlDef.h]: I think doesnt' working it's aarch64 defined and not arch64 typo

Comment (jpd002) [reply include/opengl/OpenGlDef.h]: Would make sense. @Specht2010 does that seem sensible?

Comment (Specht2010) [reply include/opengl/OpenGlDef.h]: Ah yes that's definitely a typo, sorry.  It should be aarch64.

Comment (jpd002) [reply include/opengl/OpenGlDef.h]: Thanks! Made the change. https://github.com/jpd002/Play--Framework/commit/f9209d7d52df1b91976287256af60b00a38b26ae

**Pull request merged**